oops/surface/__init__.py

```diff
@@ -18,7 +18,6 @@ class Surface(object):
     necessarily rectangular, in which two primary coordinates define locations
```

Comment:
This is the erroneous version of surface/__init__.py with all the blank lines remove. Please replace with the previous version. This file should not have any changes.